Dockerfile

```diff
@@ -29,6 +29,20 @@ RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/s
 RUN apk add gcompat
 RUN apk add jq
 
+# Add kubectl to run policies checks
+ARG KUBECTL_VERSION TARGETARCH KUBECTL_SUM_amd64 KUBECTL_SUM_arm64
```

Comment:
We should also set these ARGs in the https://github.com/aquasecurity/kube-bench/blob/main/.github/workflows/publish.yml .

Reply:
Could you provide some guidance here ? Should I declare them as `env` then as `build-args` ?

Reply:
> Should I declare them as env then as build-args ?
Yes, you are right. 

BTW, We should take care about linux/ppc64le and linux/s390x, and skip install kubectl If it doesn't support these arches. How about move the download and checksum actions to the `build` stage, then copy the binary to the main stage?